src/commands/login.ts

```diff
@@ -12,17 +14,47 @@ export default class Login extends Command {
   }
 
   async run() {
-    const email = await cli.prompt("Email", { type: "normal" })
-    const password = await cli.prompt("Password", { type: "hide" })
+    await cli.anykey("Ready! Press any key to initiate authorization flow")
 
-    this.log(`Authenticating against stagingapi.artsy.net for ${email}...`)
+    cli.action.start("Waiting for response")
 
-    const result = await new Gravity().getAccessToken({
-      email,
-      password,
-    })
+    const server = require("http")
```

Comment:
> Ah, cool -- spinning up a server to handle the callback is neat.

Yeah, I agree! Apparently this is pretty common for CLIs that initiate OAuth flows!

Based on this example: https://youtu.be/nbBqfagVdw8?t=1983

> Do we need to clean up this server at the end of this command, or does that happen for us?

As is, this spins up a one-time use server. On the first request it will shut down the server and process the input, which you can see below on L29-31

https://github.com/artsy/artsy-cli/pull/62/files#diff-f770fcb31e1ca2693fd54adb15e6e9a44e23be978206f3b79f2abbe6ab828148R29-R31

```typescript
req.connection.end()
req.connection.destroy()
server.close()
```

Reply:
ah, I missed that. 👍